specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -17264,6 +17556,14 @@
           "items": {
             "$ref": "#/definitions/ErrorEntity"
           }
+        },
+        "code": {
```

Comment:
Do we have a reason to shuffle the properties order?

Reply:
ErrorEntity was incorrectly under properties property. ARM expects it in the envelope.